apps/frontend/src/components/global/UserModal.vue

```diff
@@ -7,27 +7,29 @@
     </template>
 
     <v-card>
-      <v-card-title class="headline grey" primary-title>{{
+      <v-card-title id="userModalTitle" class="headline grey" primary-title>{{
         title
       }}</v-card-title>
       <v-card-text v-if="userInfo === null">
         <v-progress-linear indeterminate color="white" class="mb-0" />
       </v-card-text>
       <v-card-text v-else>
         <br />
-        <v-form>
+        <v-form name="userInfo">
```

Comment:
I see this uses name but other spots use `data-cy`, could you clarify on when and why you chose one versus the other?